bitnami/minio/Chart.yaml

```
@@ -25,4 +25,4 @@ name: minio
 sources:
   - https://github.com/bitnami/bitnami-docker-minio
   - https://min.io
-version: 10.0.4
+version: 10.0.5
```

```suggestion
version: 10.1.0
```

A new minor since we are adding a feature.

---

Done